tests/Kron.py

```diff
@@ -389,6 +486,71 @@ def getTolFlux(self, a, b, kfac):
 
         return tol
 
+    def testForced(self):
+        """Check that forced photometry works in the presence of rotations and translations"""
+        kfac = 2.5
+        warper = afwMath.Warper("lanczos4")
+        a = 13
+        for axisRatio in (0.25, 1.0):
+            b = a*axisRatio
+            for theta in (0, 30, 45):
+                width, height = 256, 256
+                center = afwGeom.Point2D(0.5*width, 0.5*height)
+                original = makeGalaxy(width, height, 1000.0, a, b, theta)
+                msConfig = makeSourceMeasurementConfig(kfac=kfac)
+                source = measureFree(original, center, msConfig)
+                if source.get("extensions_photometryKron_KronFlux_flag"):
+                    continue
+                angleList = [45, 90,]
+                scaleList = [1.0, 0.5]
+                offsetList = [(1.23, 4.56), (12.3, 45.6)]
+
+                for angle in angleList:
+                 for scale in scaleList:
+                  for offset in offsetList:
+#angle, scale, offset in itertools.product(angleList, scaleList, offsetList):
```

Comment:
The `product` has been split into separate `for` loops.  Note that the indentation is wrong, and splitting it up like this is unnecessary because the individual elements aren't used separately.